torch/csrc/jit/passes/onnx.cpp

```diff
@@ -249,8 +270,8 @@ void BlockToONNX(
 
   // Clone the node and add it to the new graph
   auto cloneNode = [&](Node* node) {
-    auto n_ = ctx.block->appendNode(
-        ctx.block->owningGraph()->createClone(node, envFn));
+    auto n_ = new_block->appendNode(
+        new_block->owningGraph()->createClone(node, envFn));
     for (size_t i = 0; i < node->outputs().size(); i++) {
       // n_->outputs()[i]->setType(node->outputs()[i]->type());
       env[node->outputs()[i]] = n_->outputs()[i];
```

Comment:
nit: some inconsistencies, using both outputs().at(i) and outputs()[i]

Reply:
i discovered that we could use `output(i)` as it turns out to be a checked access, calling `outputs_.at(i)` underneath. i'm updating this part with this change.